packages/docusaurus-init/templates/classic/docs/doc1.md

```diff
@@ -171,12 +171,32 @@ This line is also a separate paragraph, but... This line is only separated by a
 
 ## Admonitions
 
-:::note This is a note :::
+:::
 
-:::tip This is a tip :::
+note This is a note
 
-:::important This is important :::
+:::
 
-:::caution This is a caution :::
+:::
 
-:::warning This is a warning :::
+tip This is a tip
+
+:::
+
+:::
```

Comment:
Why not like this?

```
:::important

This is important

:::
```

Reply:
🤦‍♂ the regex I ran was wrong. I'll fix that